core/src/main/antlr4/com/databricks/labs/remorph/parsers/snowflake/SnowflakeParser.g4

```diff
@@ -1601,10 +1600,10 @@ callerOwner: CALLER | OWNER
 executaAs: EXECUTE AS callerOwner
     ;
 
-procedureBody: string
+procedureBody: (let | call | executeImmediate | string)*
```

Comment:
We should start grouping things like `let`, `call`, etc under a `statement` rule (similar to what is done with `expr`.